docs/Installation.md

```diff
@@ -390,7 +405,17 @@ See appendices for [MariaDB][MariaDB instructions for FreeBSD] and
 [PostgreSQL][PostgreSQL instructions for FreeBSD].
 
 
-### 5.3 Service startup (FreeBSD)
+### 5.3 Database configuration (FreeBSD)
+
+Finally initialize the database:
+
+```sh
+cd `perl -MFile::ShareDir -le 'print File::ShareDir::dist_dir("Zonemaster-Backend")'`
+su -m zonemaster -c "perl create_db.pl"
+```

```

Comment:
Suggestion to make it easier to just copy and paste:

1. Make `create_db.pl` executable
2. Make the command a one-liner:
``` sh
su -m zonemaster -c "$(perl -MFile::ShareDir -le 'print File::ShareDir::dist_dir("Zonemaster-Backend")')/create_db.pl"
```

Reply:
done

Reply:
I'm sorry, for FreeBSD it was a bad suggestion from me. Default shell for FreeBSD root is C shell (or tcsh), and that does not support "$( )". Bourne shell (or sh) is the default shell for other users, and that supports "$( )". Back ticks have to be used, and that affects <"> too.

``` sh
su -m zonemaster -c "`perl -MFile::ShareDir -le 'print File::ShareDir::dist_dir(qw(Zonemaster-Backend))'`/create_db.pl"
```

Reply:
This should be fixed (and I realized that I forgot to remove the call to perl since the script is now executable).